tt_metal/impl/dispatch/command_queue_interface.hpp

```diff
@@ -15,6 +15,23 @@
 
 #include <magic_enum.hpp>
 
+namespace {
```

Comment:
why duplicated?  this could live at the llrt level (basically a low level utility)